sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/CommandPreprocessor.scala

```diff
@@ -337,13 +255,6 @@ private[lf] final class CommandPreprocessor(
           contractKey,
           choiceId,
           argument,
-          strictArgument = true,
```

Comment:
This made me realize that, if I understand correctly, we're now also more lenient for arguments during replay in certain cases. Is that ok?

Reply:
In 2.x, i do not think it is a big risk taht the predecessor is bit lenient.
In fact, it is already lenient from some other aspects (e.g. it does not require map to me normalize, ...)
In 3.x, I have done a dedicate [PR](https://github.com/digital-asset/daml/pull/20036) to enforce normalization.